<launch>

  <arg name="robot_name" default="r1" />
  <arg name="scan" default="front_laser/scan" />
  
  <node pkg="mrpt_localization" type="mrpt_localization_node" name="mrpt_localization_node" output="screen">
      <remap from="scan" to="$(arg scan)"/>
      <remap from="map" to="/map"/> 
      <param name="global_frame_id" value="map"/>   
      <param name="odom_frame_id" value="$(arg robot_name)/odom"/>   
      <param name="base_frame_id" value="$(arg robot_name)/base_link"/>   
      <param name="ini_file" value="$(find mrpt_localization)/tutorial/pf-localization.ini"/> 
      <param name="map_file" value="$(find mrpt_localization)/tutorial/map.simplemap"/> 
      <param name="map_update_skip" value="3"/> 
      
  </node>

</launch>
